Sources/Node/SchemaWrapper/SchemaWrapper.swift

```diff
@@ -0,0 +1,17 @@
+public protocol SchemaWrapper:
+    NodeConvertible,
+    PathIndexable,
+    Polymorphic,
+    Equatable,
+    ExpressibleByNilLiteral,
+    ExpressibleByBooleanLiteral,
+    ExpressibleByIntegerLiteral,
+    ExpressibleByFloatLiteral,
+    ExpressibleByStringLiteral,
+    ExpressibleByArrayLiteral,
+    ExpressibleByDictionaryLiteral
+{
+    var schema: Schema { get set }
```

Comment:
Schema refers to the _structure_ of a set of data, not the data inside. This makes calling `node.schema` to get access to the underlying data confusing. I would expect `node.schema` to return something like `Dictionary` if it was holding a dictionary.

Something like `Value` or `Wrapped` would be more intuitive.

```swift
print(node.value)
// or
print(node.wrapped)
```

It's worth noting Swift.Optional does a similar thing here, and uses the word `Wrapped` 
https://github.com/apple/swift/blob/master/stdlib/public/core/Optional.swift#L122